client/netdb-white.txt

```diff
@@ -6,7 +6,7 @@
 24.234.35.55:31334
 120.79.95.7:12563
 139.99.101.187:13655
-52.191.198.174:13655
+40.71.5.58:13655
```

Comment:
You have to ask pool owners to update your IP too. Change in the github does not lead to updating whitelists on the servers.